staging/src/k8s.io/pod-security-admission/test/run.go

```diff
@@ -251,8 +257,18 @@ func Run(t *testing.T, opts Options) {
 						return
 					}
 				}
+
 				if expectSuccess && len(warningText) > 0 {
-					t.Errorf("%d: unexpected warning creating %s: %v", i, toJSON(pod), warningText)
+					permitted := false
```

Comment:
also want to make sure warningText doesn't contain policy.UnknownForbiddenReason or expectErrorSubstring which would indicate that the policy check contributed to the warning